packages/components/psammead-bulletin/src/index.test.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin } from '@bbc/gel-foundations/scripts';
+import { Link, LiveLabel } from '@bbc/psammead-story-promo';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import Image from '@bbc/psammead-image';
+import {
+  Bulletin,
+  BulletinHeading,
+  BulletinSummary,
+  BulletinCTA,
+} from './index';
+
+/* eslint-disable react/prop-types */
+const BulletinComponent = ({ script, service, isLive, dir, type, ctaText }) => {
+  const summaryText = 'This is the summary text';
+  const headlineText = 'This is the headline';
+  const imgSrc =
+    'https://ichef.bbci.co.uk/news/660/cpsprodpb/11897/production/_106613817_999_al_.jpg';
+  const bulletinLink = 'https://bbc.co.uk';
+  const imgAltText = 'Iron man';
+
+  return (
+    <Bulletin>
```

Comment:
After having a discussion with @sareh and @tochwill, we think that in this particular situation, since the Bulletin is going to have a fixed static layout, it is better we compose it in the `index.jsx` (e.g `BulletinPromo`), so we can easily import it in the stories and later on in Simorgh. Basically what @AlistairGempf initially did.

Also, it would be nice to have a couple of stories just for the `PlayCTA` component for audio and video and when it is live or not.

Reply:
If it will be static it makes sense to compose it in `index.js`.
> Also, it would be nice to have a couple of stories showing the PlayCTA component for audio and video and when it is live or not.

I don't this is necessary, we already showing it as part of the `Bulletin`
<img width="1046" alt="Screenshot 2019-10-30 at 21 41 33" src="https://user-images.githubusercontent.com/48688870/67888837-dcb72b00-fb5e-11e9-95e6-2f90cc739fe5.png">

Reply:
> I don't think this is necessary, we are already showing it as part of the Bulletin
I am happy either way 👍